### PR TITLE
Notes of SARIF TC meeting 2022-JUL-28

### PR DESCRIPTION
**Pull request by sthagen**
- [x] draft template (reflecting the tentative agenda)
- [x] ongoing meeting notes
- [x] ready to review

### CODE REVIEW

**Comment by michaelcfanning** [comment]
@sthagen @dmk42

**Comment by michaelcfanning** [comment]
Stefan please feel free to tinker/merge as necessary.

File: meeting_minutes/220728_SARIF_TC_63.md [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+
+## 6.2 Review of Decisions Reached (Co-Chair David)
+
+## 6.3 Review of Action Items (Co-Chair David)
+
+* ACTION: Mike to get someone scheduled for OpenSSF
+* ACTION: Mike: Rename from master to main, URL
+* ACTION: Mary: Continue to develop fixes scenario.
+* ACTION: Mary: Web site changes finalized and shipped.
+* ACTION: Dvid: Keep pushing to close on the wikipedia page
```
Comment (dmk42) [inline meeting_minutes/220728_SARIF_TC_63.md:127]: Dvid -> David

File: meeting_minutes/220728_SARIF_TC_63.md [hunk 2]
```diff
@@ -0,0 +1,137 @@
+# 1. Opening Activities
+
+## 1.1 Opening comments (Co-Chair David)
+
+## 1.2 Introduction of participants/roll call (Co-Chair David)
+
+| First Name | Last Name | Company           | Role(s)                 |
+|:-----------|:----------|:------------------|:------------------------|
+|            |           |                   |                         |
+Jeff Williams, Mary Martin, David Keaton, Thanassis Avgerinos, Pavel Mikula, Paul Anderson, Yekaterina O'Neil, Chris Wysopal, Nathan Baird, Michael Fanning, Aditya Sharad, Jerry Sullivan (MicroFocus) attending.
```
Comment (dmk42) [inline meeting_minutes/220728_SARIF_TC_63.md:10]: All were voting except Aditya and Jerry.  (Aditya gained voting privileges at the end of the meeting, as correctly recorded in section 1.7.2.)

**Review by dmk42 — APPROVED** [review]
Approved with minor updates.

**Comment by sthagen** [comment]
@michaelcfanning I did some tinkering @dmk42 and some fixes (member status and typo) - will merge now so we can prepare the next meeting that starts in less than ten hours.

**Comment by dmk42** [comment]
Thanks, @sthagen !

**Pull request merged**